pkg/util/mount/mount_windows.go

```diff
@@ -452,9 +450,23 @@ func (mounter *Mounter) GetFSGroup(pathname string) (int64, error) {
 	return 0, nil
 }
 
+func (mounter *Mounter) GetMode(pathname string) (os.FileMode, error) {
+	info, err := os.Stat(pathname)
```

Comment:
evalsymlinks?

Reply:
Symlinks are evaluated by the OS.

Reply:
Good point, I removed it from mount_linux